kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build-model-image.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# 
+#  Summary:
+#
+#    This script builds a model image using the WebLogic Image Tool. The
+#    tool pulls a base image if there isn't already a local base image.
+#    This script, by default, builds the model image with model files from
+#    './stage-model-image.sh' plus tooling downloaded by './stage-tooling.sh.'.
```

Comment:
Let's not have a default, but have a required argument for the location of the model file.  I'll look at stage-model-image.sh, but it's unclear why I should need to stage the model file.  At this point, this seems unnecessary.

Reply:
Staging them gives the opportunity to customize the source image on a per image basis. The demo is intended to help show the development of different images.   Different images will commonly want to different model files/archives.